plugins/action/zos_copy.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) IBM Corporation 2019-2023
+# Copyright (c) IBM Corporation 2019, 2023
```

Comment:
Still need to update the copyright year to 2024 here

Reply:
Done